### ISSUE TITLE
hasReferenceSourceDocumentType and hasReferencSourceType have wrong rdfs:range

### ISSUE THREAD

**Issue by tw-osthus** [edmcouncil/idmp#344]
```
                <owl:ObjectProperty rdf:about="&idmp-sub;hasReferenceSourceDocumentType">
                                <rdfs:subPropertyOf rdf:resource="&cmns-cls;isClassifiedBy"/>
                                <rdfs:label>has reference source document type</rdfs:label>
                                <rdfs:domain rdf:resource="&idmp-sub;ReferenceSourceDocument"/>
                                <rdfs:range rdf:resource="&idmp-sub;hasReferenceSourceDocumentType"/>
                </owl:ObjectProperty>
                
                <owl:ObjectProperty rdf:about="&idmp-sub;hasReferenceSourceType">
                                <rdfs:subPropertyOf rdf:resource="&cmns-cls;isClassifiedBy"/>
                                <rdfs:label>has reference source type</rdfs:label>
                                <rdfs:domain rdf:resource="&idmp-sub;ReferenceSource"/>
                                <rdfs:range rdf:resource="&idmp-sub;hasReferenceSourceType"/>
                </owl:ObjectProperty>
```
should be ReferenceSourceDocumentType instead of hasReferenceSourceDocumentType resp. ReferenceSourceType instead of hasReferenceSourceType

**Comment by tw-osthus** [comment]
found another one
```
<owl:Class rdf:about="&idmp-sub;ReferenceSource">
		<rdfs:subClassOf rdf:resource="&cmns-doc;Reference"/>
		<rdfs:subClassOf>
			<owl:Restriction>
				<owl:onProperty rdf:resource="&idmp-sub;hasReferenceSourceURL"/>
				<owl:onClass rdf:resource="&xsd;anyURI"/>
				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
			</owl:Restriction>
		</rdfs:subClassOf>
...
		<rdfs:subClassOf>
			<owl:Restriction>
				<owl:onProperty rdf:resource="&idmp-sub;hasCorrespondingDocument"/>
				<owl:onClass rdf:resource="&xsd;anyURI"/>
				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
			</owl:Restriction>
		</rdfs:subClassOf>
```

should be owl:onDataRange instead of owl:onClass

**Comment by tw-osthus** [comment]
fixed it in branch bugfix-344

**Comment by ElisaKendall** [comment]
This issue was resolved by pull request #345
